### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the edit fields  in "Run on web App for Containers" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1464]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Run on web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to and select  "web app for container" button.->create new radio button
6. Tab to Docker File , Server URL ,Username ,Image and Tag and Start up file  edit boxes.
    App name edit box
    Create new edit boxes under "Resource group" expand button and "App Service Plan" expand button .
7. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA is not announcing the name of the edit box

**Jaws behaviour:**
Edit
C:/Users/abby/IdeaProjects/untitled4/untitled4.iml
Type in text
Edit
wiprosample1.azurecr.io
Type in text
Edit
wiprosample1
Type in text
Edit
image:tag
Type in text

**NVDA Behaviour:** 
edit
blank
edit
blank
edit
blank
edit
blank
edit
rg-web-linux-180208010817
edit
appsp-linux--180208010817

**Expected Behaviour:**
Jaws and NVDA should announce the name of the edit box
Name should be
Docker File
Server URL
Username
Image and Tag
Start Up File
App Name
Create new edit boxes under "Resource group" expand button and "App Service Plan" expand button

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for container-edit-name -empty](https://user-images.githubusercontent.com/22956130/36102039-0d1c673a-1031-11e8-98ac-384e5d012e4b.jpg)
![d1763a97](https://user-images.githubusercontent.com/22956130/36102045-10555934-1031-11e8-9526-490b3a7538c1.jpg)
![run on web app for containers-edit-explorer](https://user-images.githubusercontent.com/22956130/36102051-12b27dd8-1031-11e8-85d7-91ba013e76c5.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749928
